lib/streamlit/config.py

```diff
@@ -185,11 +184,24 @@ def _delete_option(key):
         pass
 
 
+# Config Section: Mapbox #
+
+_create_section("mapbox", "Mapbox configuration that is being used by DeckGL.")
+
+
+_create_option("mapbox.token",
+                description="""Mapbox uses access tokens to associate your 
+                account with your requests to Mapbox API resources.
```

Comment:
How about:
> Configure Streamlit to use a custom Mapbox token for elements like st.deck_gl_chart and st.map. If you don't do this you'll be using Streamlit's own token, which has limitations and is not guaranteed to always work. To get a token for yourself, create an account at https://mapbox.com. It's free! (for moderate usage levels)

Reply:
We should also add some similar text to the st.deck_gl_chart and st.map docstrings. Something like:

> When using this method, we advise all users to use a personal Mapbox token. This ensures the map tiles used in this chart are more robust. You can do this with the `mapbox.token` config option.
> *  To get a token for yourself, create an account at https://mapbox.com. It's free! (for moderate usage levels)
> * See https://streamlit.io/docs/cli.html#view-all-config-options for more info on how to set config options.

Reply:
Should we add a warning in the frontend?